crates/binstalk-git-repo-api/Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "binstalk-git-repo-api"
+description = "The binstall toolkit for accessing API for git repository"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+documentation = "https://docs.rs/binstalk-git-repo-api"
+version = "0.1.0"
```

Version should be 0.0.0, so that we can cut a new 0.1.0 release

```suggestion
version = "0.0.0"
```